scripts/tools/bouffalolab/factory_qrcode.py

```diff
@@ -20,13 +20,13 @@
 
 try:
     import qrcode
-    from SetupPayload import CommissioningFlow, SetupPayload
+    from matter_setup_payload import CommissioningFlow, SetupPayload
```

Comment:
Would it be possible to install this package in the `chip.setup_payload` namespace? Currently we have a `setup_payload` module available here `src/controller/python/chip/setup_payload/__init__.py`. Anyway, I'm not sure what is the difference between this new package and current `from chip.setup_payload import ...`. The case is that it would be nice (IMHO) to have all Python packages created by matter SDK repo available under a common namespace, i.e.: `chip.` (since we have this namespace now).

Reply:
`src/controller/python/chip/setup_payload` is a wrapper on top of C++ code, and this is written in Python.

> from chip.setup_payload import .... The case is that it would be nice (IMHO)

Agreed, will restructure.

> If there is something common

Both functionally do the same thing, and yes, some utilities which only requires generating onboarding payload will not need the complete chip-core. So, this has to be split out.